src/it/default-fork/src/test/java/MyTest.java

```diff
@@ -18,6 +18,4 @@
  */
 package foo;
 
-import junit.framework.TestCase;
```

Comment:
Same comment as above about keeping something (anything) available only under the `test` scope, for verifying that the compiler put the correct dependencies on the classpath.